www/vokp00/cv05hw/css/styles.css

```diff
@@ -0,0 +1,125 @@
+body {
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.div1 {
+    color: red;
+}
+
+.div2 {
+    color:  rgb(34,139,34);
+}
+
+.div3 {
+    color:  #4169E1;
+    font-weight: bold;
+}
+
+.div4 {
+    font-size:  18px;
+}
+
+.div5 {
+    font-size:  24px;
+    font-style: italic; 
+}
+
+.div6 {
+    color:  white;
+    background-color: brown; 
+}
+
+.p1 {
+    text-align: center;
+    line-height: 32px;
+}
+
+.p2 {
+    text-transform: uppercase;
+}
+
+.box1 {
+    width: 340px;
+    height: 240px;
+    border: 1px blueviolet solid;
+    padding: 30px;
+}
+
+.box2 {
+    width: 500px;
+    height: 100px;
+    background-color: red;
+    margin-left: 100px;
+    margin-right: 100px;
+}
+
+.box3 {
+    width: 100%-100px;
+    height: 80px;
+    background-color: blue;
+    padding: 50px;
+    border: 10px;
+    border-style: solid;
+    border-color: yellow;
+}
+
+.box4 {
+    width: 200px;
+    height: 500px;
+    background-color: green;
+    opacity: 0.5;
+}
+
+.box5 {
+    width: 400px;
+    height: 300px;
+    position: relative;
+    background-color: lightblue;
+}
+
+.box6 {
+    font-size: 4px;
+    width: 50px;
+    height: 50px;
+    background-color: pink;
+    position: absolute;
+    top: 50px;
+    right: 10px;
+}
+
+.box7 {
+    width: 200px;
+    height: 500px;
+    background-color: blue;
+    margin-top: 100px;
+}
+
+.box7:hover {
+    cursor: pointer;
```

Comment:
cursor: pointer nemusi byt v :hover, protoze jiz implikuje to ze musi byt hover aby to melo efekt